ledger/src/blockstore_meta.rs

```diff
@@ -710,6 +716,27 @@ impl ErasureMeta {
         coding_shred.check_coding_shred(shred2)
     }
 
+    /// Returns true if `index` and `fec_set_index` are valid under the assumption that
+    /// all erasure sets contain exactly `DATA_SHREDS_PER_FEC_BLOCK` data and coding shreds:
+    /// - `index` is between `fec_set_index` and `fec_set_index + DATA_SHREDS_PER_FEC_BLOCK`
+    /// - `fec_set_index` is a multiple of `DATA_SHREDS_PER_FEC_BLOCK`
+    pub fn check_fixed_fec_set(index: u32, fec_set_index: u32) -> bool {
```

Comment:
should we not have this together with basic shred format validation outside of blockstore code?

Reply:
yeah good call [8622835](https://github.com/anza-xyz/agave/pull/7728/commits/8622835d922f609064b63d14bf1e3916d25c2f6f)